src/components/skeletons/SourceSkeleton.tsx

```diff
@@ -0,0 +1,11 @@
+import Skeleton from "react-loading-skeleton";
+
+export default function SourceSkeleton() {
+  return (
+    <div className="w-full grid grid-flow-col  gap-2  sm:gap-5">
+      {[0, 1, 2, 3, 4, 5, 6, 7, 8].map((_, index) => (
```

Comment:
You can add type on this map argument because right now it can be anything 
```suggestion
      {[0, 1, 2, 3, 4, 5, 6, 7, 8].map((_, index:number) => (
```